docs/sources/whatsnew/whats-new-in-v10-2.md

```diff
@@ -0,0 +1,52 @@
+---
+description: Feature and improvement highlights for Grafana v10.2
+keywords:
+  - grafana
+  - new
+  - documentation
+  - '10.2'
+  - release notes
+labels:
+products:
+  - cloud
+  - enterprise
+  - oss
+title: What's new in Grafana v10.2
+weight: -39
+---
+
+# What’s new in Grafana v10.2
+
+Welcome to Grafana 10.2! Read on to learn about changes to ...
+
+For even more detail about all the changes in this release, refer to the [changelog](https://github.com/grafana/grafana/blob/master/CHANGELOG.md). For the specific steps we recommend when you upgrade to v10.2, check out our [Upgrade Guide]({{< relref "../upgrade-guide/upgrade-v10.2/index.md" >}}).
+
+<!-- Template below
+
+> Add on-prem only features here. Features documented in the Cloud What's new will be copied from those release notes.
+
+## Feature
+<!-- Name of contributor -->
+<!-- _[Generally available | Available in private/public preview | Experimental] in Grafana [Open Source, Enterprise]_
+Description. Include an overview of the feature and problem it solves, and where to learn more (like a link to the docs).
+{{% admonition type="note" %}}
+You must use relative references when linking to docs within the Grafana repo. Please do not use absolute URLs. For more information about relrefs, refer to [Links and references](/docs/writers-toolkit/writing-guide/references/).
+{{% /admonition %}}
+-->
+<!-- Add an image, GIF or video  as below
+
+{{< figure src="/media/docs/grafana/dashboards/WidgetVizSplit.png" max-width="750px" caption="DESCRIPTIVE CAPTION" >}}
+
+Learn how to upload images here: https://grafana.com/docs/writers-toolkit/write/image-guidelines/#where-to-store-media-assets
+-->
+
+## Authentication and authorization
+
+### No Basic Role
+Generally available in all editions of Grafana
+
+Introducing a new basic role with no permissions, called "No Basic Role." This role is perfect for when users would like their service accounts or users to have tailored customized RBAC permissions. This can be set as a basic role via the API or UI.
```

Comment:
@eleijonmarck couple of thoughts:

- I think readers would benefit from having references to some of the definitions, such as what is a basic role and a permission. 
- If the feature is GA, there should not be a feature toggle. Can you please check and either remove the toggle, or update to the correct availability note?
- For the what's new content, might be useful to have a note around what was previously and what users have now. 
- I'd remove the note about what is important to remember, and link to the respective docs instead. 


Also, would appreciate if you can send a new PR with the changes, instead of changing directly here. Thank you!